components/collective-page/hero/HeroEventDetails.js

```diff
@@ -135,7 +135,7 @@ class HeroEventDetails extends React.Component {
         {location.name && (
           <HeroNote>
             <MapPin size={16} />
-            <Link route={locationRoute}>
+            <Link href={`/${locationRoute}`}>
```

Comment:
No need for the `#` here, locationRoute is a hash: https://github.com/opencollective/opencollective-frontend/blob/55b9d3f688e80e01b944b5df6f0b9fc950770490/components/collective-page/hero/HeroEventDetails.js#L89

```suggestion
            <Link href={`${locationRoute}`}>
```